templates/patterns/toasts/toasts.js

```diff
@@ -0,0 +1,11 @@
+Drupal.behaviors.toasts = {
+  attach: function () {
+    var toastElList = [].slice.call(document.querySelectorAll(".toast"));
+    var options = { autohide: false };
```

Comment:
The specification of autohide for toasts is here: https://getbootstrap.com/docs/5.0/components/toasts/

As specified there, the autohide: false is needed because : "Toasts will automatically hide if you do not specify autohide: false."

So for the moment we can leave it like that.